### PR TITLE
Add configurable ADB commands timeout. Fixes #7.

### LINKED ISSUE #7: Increase ADB install timeout or make it configurable
Timeout for APK installation is set to only 5 minutes:
https://github.com/gojuno/commander/blob/53236f00b4d7dd9e07246f3c8eb961739188dede/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt#L61

I have a ~200 MB APK and a device connected remotely and 5 minutes is not enough.
In ddmlib (so also in spoon) there is a deafult timeout set to 5 s, in case of installation it seems to refer to a [single 64 KB chunk of data](https://android.googlesource.com/platform/tools/base/+/tools_r22/ddmlib/src/main/java/com/android/ddmlib/SyncService.java#671). Additionally default value can be increased using `com.android.ddmlib.DdmPreferences#setTimeOut`.

`adb install` constantly prints percentage during installation. Maybe installation timeout should occur when no new output is received within some time period e.g. 30 s?

### PR DESCRIPTION
**Pull request by koral--**
If this is merged I'll fill PR in Composer.

### CODE REVIEW

File: android/src/main/kotlin/com/gojuno/commander/android/Adb.kt [hunk 1]
```diff
@@ -53,12 +54,12 @@ fun connectedAdbDevices(): Observable<Set<AdbDevice>> = process(listOf(adb, "dev
 
 fun AdbDevice.log(message: String) = com.gojuno.commander.os.log("[$id] $message")
 
-fun AdbDevice.installApk(pathToApk: String): Observable<Unit> {
+fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 5 to MINUTES): Observable<Unit> {
```
Comment (artem-zinnatullin) [inline android/src/main/kotlin/com/gojuno/commander/android/Adb.kt]: Let's reduce default to 2 minutes, sounds reasonable?

**Review by artem-zinnatullin — APPROVED** [review]
Looks great!

File: android/src/main/kotlin/com/gojuno/commander/android/Adb.kt (hunk 1 above)
Comment (koral--) [reply android/src/main/kotlin/com/gojuno/commander/android/Adb.kt]: OK.
Both composer and commander are before version 1.0.0 so I assume that it is acceptable that users which currently need timeout between 2 and 5 minutes will have to adjust their configurations.

Comment (artem-zinnatullin) [reply android/src/main/kotlin/com/gojuno/commander/android/Adb.kt]: yep, that's the rationale, once we get to 1.x will strictly follow semantic versioning!

**Review by yunikkk — APPROVED** [review]
TY!

**Pull request merged**